helpers.js

```diff
@@ -381,11 +381,24 @@ function doReport(config, objects, errors, notAnalyzedContracts) {
     }
 
     const logs = objects.map(obj => obj.logs)
-        .reduce((acc, curr) => acc.concat(curr), []);
-    if (logs.length > 0) {
+          .reduce((acc, curr) => acc.concat(curr), []);
+
+    // Ignore logs with log.level "info" unless the "debug" flag
+    // has been set.
+    let haveLogs = false;
+    for(const log of logs) {
+        if (!config.debug && log.level !== 'info') {
```

Comment:
Does this `log.level` come from HTTP response of Mythril API ?
If it is true, LGTM.
Thx!

Reply:
Mythril API returns strings. We had it returning other things like errors, but we were getting useless tracebacks. We don't want HTTP status code either. Strings are simple.